assets/src/edit-story/components/checklist/checks/storyPosterAttached.js

```diff
@@ -34,46 +42,32 @@ export function storyHasNoPosterAttached(story) {
 export function StoryPosterAttached() {
   //@TODO refine this context selector and storyHasNoPosterAttached to run more selectively
   const story = useStory(({ state }) => state);
-  return storyHasNoPosterAttached(story)
-    ? // <ChecklistCard
-      //   title={__('Add Web Story poster image', 'web-stories')}
-      //   titleProps={{
-      //     onClick: () => {
-      //       /* perform highlight here */
-      //     },
-      //   }}
-      //   footer={
-      //     <Text size={THEME_CONSTANTS.TYPOGRAPHY.PRESET_SIZES.X_SMALL}>
-      //       <ul>
-      //         <li>
-      //           {__('Use as a representation of the story.', 'web-stories')}
-      //         </li>
-      //         <li>{__('Avoid images with embedded text.', 'web-stories')}</li>
-      //         <li>
-      //           {__("Use an image that's at least 640x853px.", 'web-stories')}
-      //         </li>
-      //         <li>{__('Maintain a 3:4 aspect ratio.', 'web-stories')}</li>
-      //       </ul>
-      //       <Link
-      //         href={'#' /* figure out what this links to */}
-      //         size={THEME_CONSTANTS.TYPOGRAPHY.PRESET_SIZES.X_SMALL}
-      //       >
-      //         {'Learn more'}
-      //       </Link>
-      //     </Text>
-      //   }
-      //   cta={
-      //     <Button
-      //       size={BUTTON_SIZES.SMALL}
-      //       type={BUTTON_TYPES.SECONDARY}
-      //       onClick={() => {
-      //         /* perform highlight here */
-      //       }}
-      //     >
-      //       {__('Upload', 'web-stories')}
-      //     </Button>
-      //   }
-      // />
-      null
-    : null;
+  const setHighlights = useHighlights(({ setHighlights }) => setHighlights);
+
+  return storyHasNoPosterAttached(story) ? (
+    <ChecklistCard
+      title={PRIORITY_COPY.storyMissingPoster.title}
+      footer={
+        <FooterText>
+          <CardListWrapper>
+            <List size={THEME_CONSTANTS.TYPOGRAPHY.PRESET_SIZES.X_SMALL}>
+              {PRIORITY_COPY.storyMissingPoster.footer}
+            </List>
+            <LearnMoreLink />
+          </CardListWrapper>
+        </FooterText>
+      }
+      cta={
+        <DefaultCtaButton
+          onClick={() =>
+            setHighlights({
+              highlight: states.POSTER,
+            })
+          }
+        >
+          {__('Upload', 'web-stories')}
```

Comment:
🤔  should we (eventually not here) add the CTA text to the copy for the issue?

Reply:
I figured for something this small it wasn't really that related to the check. Happy to put it in the `_COPY` constants tho. makes sense as part of the check

Reply:
we can do that now or later, mostly thinking out loud.